layouts/partials/docs/toc.html

```diff
@@ -1,21 +1,50 @@
 {{ $menu := .Site.Menus.main }}
 {{ $has_toc := false }}
+
+{{ $isDocsHome := eq .RelPermalink "/docs/" }}
+{{ $isGetStarted := false }}
+{{ $isReference := false }}
+{{ $isAWS := false }}
+
 {{ if hasPrefix .RelPermalink "/docs/quickstart/" }}
     {{ $menu = .Site.Menus.quickstart }}
     {{ $has_toc = true }}
+    {{ $isGetStarted = true }}
 {{ else if hasPrefix .RelPermalink "/docs/reference/"}}
     {{ $menu = .Site.Menus.reference }}
     {{ $has_toc = true }}
+    {{ $isReference = true }}
 {{ else if hasPrefix .RelPermalink "/docs/tour/" }}
     {{ $menu = .Site.Menus.tour }}
     {{ $has_toc = true }}
 {{ else if hasPrefix .RelPermalink "/docs/aws/" }}
     {{ $menu = .Site.Menus.aws }}
     {{ $has_toc = true }}
+    {{ $isAWS = true }}
 {{ end }}
 
-{{ if $has_toc }}
-    <aside class="toc">
+<aside class="toc">
+    <h3 class="no-anchor">
+        <a href="{{ relref . "/docs" }}">Docs</a>
```

Comment:
I went back and forth on including this. Ultimately, I left it in, similar to how we have a heading for the Pulumi Blog in the left nav. But I'd be OK with getting rid of it as it's a bit redundant with the nav bar.

Reply:
Earlier when we were talking about a docs redesign, we had a lot of ideas around making the left nav more useful. So if anything, we might want the next step to be adding more stuff to the left-nav. Perhaps quick links like "Clouds" or "Languages"?

I can only guess at what that would look like, but I would just be a little hesitant to removing it.

Reply:
... If we do add more if-conditions, we probably want to initialize all those one-of flags like `$isAWS` into a map, and then do lookups. (So we can just use string constants instead of N-variables, in case that is cleaner.) e.g. `if (Get $flags "is-aws")`

Reply:
> Earlier when we were talking about a docs redesign, we had a lot of ideas around making the left nav more useful. So if anything, we might want the next step to be adding more stuff to the left-nav. Perhaps quick links like "Clouds" or "Languages"?

Yeah, as a next step, we should look at reorganizing all the docs content. There's lots of opportunities to make things more visible that are currently all buried under `/reference`. "Clouds" and "Languages" are good candidates, also "Tutorials".

> I can only guess at what that would look like, but I would just be a little hesitant to removing it.

Oh, I didn't mean removing the left nav, I just wasn't sure if we should include a "Docs" header at the top of the nav or not.